Example/Auth/ApiTests/GoogleAuthTests.m

```diff
@@ -63,12 +63,16 @@ - (void)testSignInWithGoogle {
                                            error.localizedDescription);
                                  }
                                }];
-  XCTAssertEqualObjects(auth.currentUser.displayName, kGoogleTestAccountName);
+
+  if (auth.currentUser.displayName) {
```

Comment:
See reference below (line 74-75). It seems to be an issue in practice; one of my test accounts did not have an id_token returned by the refresh flow.